components/org.wso2.carbon.identity.application.authenticator.google/src/main/java/org/wso2/carbon/identity/application/authenticator/google/Utils.java

```diff
@@ -77,6 +89,16 @@ public static boolean validateGoogleJWT(String idToken, String audience)
 
         // Verifying the issuers and audiences.
         if (claimsSet != null && claimsSet.toJSONObject() != null && !claimsSet.toJSONObject().isEmpty()) {
+
```

Comment:
We can remove this additional new line.

Reply:
Fixed 414aed35225d4d7f2cf7ef2d89e7daddc3ca3a68

Reply:
Shall we remove this redundant line?

Reply:
Fixed a25f96079ab9450d25aa080915fe5b0e454d69aa